src/main/java/bob/storage/FileReading.java

```diff
@@ -0,0 +1,80 @@
+package bob.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+import bob.data.TaskList;
+import bob.tasks.DeadlineTask;
+import bob.tasks.EventTask;
+import bob.tasks.Task;
+import bob.tasks.TodoTask;
+
+/**
+ * Class representing the file reading.
+ */
+public class FileReading extends Storage {
```

Comment:
This class can be renamed to FileReader so that it follows the coding conventions of Class names being nouns